types/test-listen/index.d.ts

```diff
@@ -0,0 +1,15 @@
+// Type definitions for test-listen 1.1
+// Project: https://github.com/zeit/test-listen
+// Definitions by: Stephen Mathieson <https://github.com/stephenmathieson>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.1
+
+/// <reference types="node" />
+
+import { Server } from "net";
+
+declare function testListen(srv: Server, hostname?: string): Promise<string>;
+
+declare namespace testListen {}
```

Comment:
It seems this empty namespace can be removed

Reply:
The purpose of the empty namespace is to allow `import * as testListen from 'test-listen'`. It's a hack. I'll remove.